stdlib/array.gr

```diff
@@ -414,3 +414,47 @@ export let join = (separator: String, items: Array<String>) => {
     Some(s) => s,
   }
 }
+
+/**
+ * Slices an array given zero-based start and end indexes. The value
+ * at the end index will not be included in the result.
+ *
+ * If either index is a negative number, it will be treated as a reverse index from
+ * the end of the array. e.g. slice(1, -1, [> 'a', 'b', 'c']) == [> 'b'].
+ *
+ * @param startIndex: Number The index of the array where the slice will begin (inclusive)
+ * @param endIndex: Number The index of the array where the slice will end (exclusive)
+ * @param array: Array<a> The array to be sliced
+ * @returns Array<a> The subset of the array that was sliced
+ */
+export let slice = (startIndex, endIndex, array) => {
+  // Note(blaine): Implementation taken directly from the String.slice method
+  // I wrote in Reason for Grain_utils.String
```

Comment:
I personally don't feel like we need this note here, but I don't feel crazy strong about it.
```suggestion
```

Alternatively, if you want to keep it, it might be better to put `phated` here instead of `blaine` since it might be easier for people to find you:
```suggestion
  // NOTE(phated): Implementation taken directly from the String.slice method
  // I wrote in Reason for Grain_utils.String
```

Reply:
I think it's fine to remove.